mix.exs

```diff
@@ -22,7 +22,8 @@ defmodule Mariaex.Mixfile do
     [{:decimal, "~> 1.0"},
      {:db_connection, "~> 1.1"},
      {:coverex, "~> 1.4.10", only: :test},
-     {:ex_doc, ">= 0.0.0", only: :dev}]
+     {:ex_doc, ">= 0.0.0", only: :dev},
+     {:poison, "~> 2.1", optional: true}]
```

Comment:
Please set version as `>= 0.0.0` to prevent overrides being required. Many users will be on either `2.0` or `3.0`.

Reply:
Sorry for missing consideration. I fixed via 0a88aea .